.github/workflows/integration-test.yml

```
@@ -144,3 +144,52 @@ jobs:
           path: dist/*.tar.gz
           if-no-files-found: error
           retention-days: 90
+  docker-image-test:
+    runs-on: ubuntu-latest
+    # set a dependency sequence to occur after build job completion
+    needs: build
+    steps:
```

```suggestion
    needs: build
    env:
      version: ${{ needs.build.outputs.version }}
      sdist_filename: pycytominer-${{ needs.build.outputs.version}}.tar.gz
    steps:
```
Recommend adding some useful env variables at the start

---

Cheers, this is a lot more clean! Thank you.